tests/python/unittest/test_loss.py

```diff
@@ -144,7 +144,7 @@ def test_kl_loss():
     assert mod.score(data_iter, eval_metric=mx.metric.Loss())[0][1] < 0.05
 
 
-@with_seed(1234)
+@with_seed()
```

Comment:
nit: please refer to https://github.com/apache/incubator-mxnet/pull/11888/files#diff-cb652780258e73a9cd08568f38929aa2R2509 for adding some tracking for this test.

Reply:
There was no report of flakiness on this test (no failures as far as I know). Using the same comment you sent is confusing. I can add a line referencing the Github issue in the test, although developers can always track changes via git blame, and the issue is referenced in the PR. I don't see it as a common practice to mention all Github issue number every time a test is updated.

Reply:
I've approved this before I sent this nit, you are free to remove it if you think it's unnecessary. I simply thought this would be a quicker way to track issues and attribute ownership in the future.